app/controllers/admin/environments_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Admin::EnvironmentsController < ApplicationController
+  before_action :authorize_admin!
+  before_action :authorize_super_admin!, only: [ :create, :new, :edit, :update, :destroy ]
+  before_action :environment, only: [:edit, :update, :destroy]
```

Comment:
is there some Rails magic where you don't explicitly have to define the default CRUD actions?